lib/MySQL_Thread.cpp

```diff
@@ -5669,6 +5935,9 @@ unsigned long long MySQL_Threads_Handler::get_total_backend_stmt_close() {
 				q+=__sync_fetch_and_add(&thr->status_variables.backend_stmt_close,0);
 		}
 	}
+	const auto& cur_val = this->status_variables.p_counter_array[p_th_counter::com_backend_stmt_close]->Value();
```

Comment:
Also here, let's use a macro.
In those lines and all the other similar in this file

Reply:
Fixed!